src/api.jl

```diff
@@ -429,7 +492,11 @@ function infer(fieldids::Vector{Tuple{Int, Int, Int}},
     # fitting the PSF for the whole catalog above)
     info("fitting PSF for all relevant sources")
     tic()
-    ModelInit.fit_object_psfs!(mp, target_sources, images)
+    if Threaded
+      fit_object_psfs_threaded!(mp, target_sources, images)
+    else
+      ModelInit.fit_object_psfs!(mp, target_sources, images)
+    end
     times.fit_psf = toq()
```

Comment:
It should be possible to have a single function ``fit_object_psfs!()`` rather than separate ones for threads and serial -- see ``infer()``. There should be no need to test ``Threaded`` because if it is disabled, things like ``@threads`` do nothing, and ``threadid()`` and ``nthreads()`` always return 1.